src/Services/Utils/SQLUpgradeService.php

```diff
@@ -1047,8 +1048,9 @@ private function CreateOccupationList()
     private function CreateReactionList()
     {
         $res = sqlStatement("SELECT DISTINCT reaction FROM lists WHERE reaction <> ''");
+        $records = [];
         while ($row = sqlFetchArray($res)) {
-            $records[] = $row['reaction'];
+            $records = $row['reaction'];
```

Comment:
isn't this supposed to create an array of reactions?
(i think these changes will just collect the last reaction as a string)